setup.py

```diff
@@ -83,7 +83,7 @@ def read(fname):
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
-    install_requires=['pytz'],
+    install_requires=['pytz', 'sqlparse'],
```

Comment:
I know historically the Python ecosystem hasn't worried much about version pinning, but it seems more things are semantically versioned these days, so shouldn't we worry about breaking changes and declare `sqlparse < 1.0.0` ?

Reply:
I think we shouldn't try to predict the future as to how sqlparse will do its versioning. Users can pin version requirements in a requirements file whereas if some restriction is here, it's difficult if not impossible to override it, I believe. Let me know if you have a difference experience.

Reply:
I'd say it's less likely to cause problems if Django pins `<1.0.0` for the time being, and then if there is a breaking release, users without very good pinning won't be affected. Django can always issue a patch release to move the pin if a new sqlparse comes out and it is compatible.

Reply:
It also says:
> It may also be known that project A follows semantic versioning, and that v2 of ‘A’ will indicate a break in compatibility, so it makes sense to not allow v2:

I guess the real question is then whether `sqlparse` really uses semantic versioning, and since it's "Development Status :: 5 - Production/Stable" at 0.2.4 potentially not.

You're probably right to err this way, let's leave it as-is. :)